4717_Sprint/9-Haoran/Demo.java

```diff
@@ -0,0 +1,61 @@
+import java.util.*;
+import java.time.*;
+import java.time.format.*;
+
+public class Demo {
+
+  private void siriMode(String input, Siri siri){
+
+    if (input.contains("sassy") && "normal".equals(siri.getChatbotMode())) {
+      siri.setChatbotMode("sassy");
+      System.out.println("SassySiri model on.");
+    } else if ("sassy".equals(siri.getChatbotMode()) && input.contains("normal") ) {
+      siri.setChatbotMode("normal");
+      System.out.println("Normal model on.");
+    }else throw new IllegalArgumentException("Siri model does not exist.");
```

Comment:
it is better to use {} for the else. Also check the missing white space at the beginning of the line